scss/blocks/figma-components.scss

```diff
@@ -0,0 +1,101 @@
+:root {
+    --checkered-canvas-bg: #fff;
+    --checkered-square: #182d660a;
+    --checkered-square-size: 8px;
+}
+
+.figma-components {
+    display: grid;
+    gap: 24px;
+    margin: 12px 0;
+}
+
+.figma-component {
+    display: flex;
+    flex-direction: column;
+    gap: 8px;
+}
+
+.figma-component-details {
+    display: flex;
+    flex-direction: column;
+    gap: 4px;
+}
+
+.figma-component-image {
+    display: flex;
+    justify-content: center;
+    padding: 8px;
+}
+
+.figma-component-image-background {
+    background-image: linear-gradient(45deg,var(--checkered-square) 25%,transparent 25%,transparent 75%,var(--checkered-square) 75%,var(--checkered-square) 100%),linear-gradient(45deg,var(--checkered-square) 25%,var(--checkered-canvas-bg) 25%,var(--checkered-canvas-bg) 75%,var(--checkered-square) 75%,var(--checkered-square) 100%);
+    background-position: 0 0,calc(var(--checkered-square-size) / 2) calc(var(--checkered-square-size) / 2);
+    background-size: var(--checkered-square-size) var(--checkered-square-size);
+}
+
+.figma-component-image-border {
+    border-radius: 5px;
+    border: 1px solid #eceef3;
+}
+
+.figma-component-image img {
+    max-height: 100%;
+    max-width: 100%;
+    object-fit: contain;
+    height: auto;
+}
+
+.figma-component-name {
+    font-size: 14px;
+    font-weight: 500;
```

Comment:
+1 to this, that applies to all across the whole file (font sizes, font weights, colors, ...), otherwise customer-overrides of the theme will not work properly

Reply:
[f9cfeab](https://github.com/Supernova-Studio/exporter-documentation/pull/167/commits/f9cfeabd9eb7e8c4af923b1177a5d70d1ee882cd)

We don't seem to have `fontSizes` and the colors that are in this design in `theme.scss` 🤔